activation/activation-privacy-statement.md

```diff
@@ -4,8 +4,8 @@ author: Matija Hanzic
 date: 16/5/2017  
 description: This article lists Acceleratio Privacy Statement.
 --- 
-Acceleratio Ltd. asks you activate your copy of SPDocKit so we may verify your product was purchased legally. During activation process your personal data, your product key is not being sent to Acceleratio Ltd.
+Acceleratio Ltd. asks you to activate your copy of SPDocKit so that we may verify your product was purchased legally. Your personal data and product key __will not be__ sent to Acceleratio Ltd.
```

Comment:
I'm not sure, it leaves open the implication this data might then be sent otherwise.